specs/phase0/fork-choice.md

```diff
@@ -623,6 +623,9 @@ def update_latest_messages(store: Store, attesting_indices: Sequence[ValidatorIn
 
 ```python
 def on_tick(store: Store, time: uint64) -> None:
+    # Precondition
+    assert time >= store.time
```

Comment:
I'm not certain we should add this unless we are going to actively test it. Our guiding principle is that asserts should be triggerable with invalid inputs for spec and testing purposes. But this would only really be triggerable if the client's system was broken rather than them getting invalid messages